packages/dso-toolkit/src/components/plekinfo-card/plekinfo-card.stories-of.ts

```diff
@@ -0,0 +1,128 @@
+import { ComponentAnnotations, PartialStoryFn, Renderer } from "@storybook/types";
+
+import {
+  PlekinfoCardArgs,
+  plekinfoCardArgs,
+  plekinfoCardArgTypes,
+  plekinfoCardArgsMapper,
+} from "./plekinfo-card.args.js";
+import { PlekinfoCard } from "./plekinfo-card.models.js";
+
+import { StoriesParameters, StoryObj } from "../../template-container.js";
+import { compiler } from "markdown-to-jsx";
+import { MetaOptions } from "../../storybook/meta-options.interface.js";
+import { plekinfoCardDemoCss } from "./plekinfo-card.demo";
+
+export type PlekinfoCardDecorator<TemplateFnReturnType> = (story: PartialStoryFn, css: string) => TemplateFnReturnType;
+
+type PlekinfoCardStory = StoryObj<PlekinfoCardArgs, Renderer>;
+
+interface PlekinfoCardStories {
+  Default: PlekinfoCardStory;
+  WithoutSymbol: PlekinfoCardStory;
+  WithLabel: PlekinfoCardStory;
+  WithNameChange: PlekinfoCardStory;
+  WithNameChangeComplex: PlekinfoCardStory;
```

Comment:
We hebben dus deze requirement: 

"Chevron nooit alleen op een regel plaatsen, indien dit het geval is het laatste woord meenemen naar de volgende regel."

En ik zie dit
<img width="1017" alt="Screenshot 2024-10-10 at 14 41 54" src="https://github.com/user-attachments/assets/136cf844-d54a-4085-aade-3fa90b66166c">

Ik ga dat even aan Django verleegen, denk ik.

Reply:
Hmm, had ik al gedaan. Zie de issue zelf.

Reply:
Dit is eigenlijk niet te doen weet ik inmiddeks, zeker niet in de huidige structuur (slot + icon). ik zou zeggen nu releasen in huidige opzet, knokken we hier later nog wel over

Reply:
Eens. We hebben dit wel eens opgelost, maar met paddings en background images. Ook weer niet echt fraai, helemaal gezien de issues met #2813.

Ik vind dit prachtig wrappen, maar dat ben ik 🤷

Reply:
Precies. Alles wat je hierop verzint is bij voorbaat een hack.

Reply:
Helemaal goed